libs/sdk-ui-kit/src/Header/tests/generateHeaderMenuItemsGroups.test.ts

```diff
@@ -233,4 +417,51 @@ describe("generateHeaderMenuItemsGroups", () => {
             ],
         ]);
     });
+
+    it("should not return load item with workspace in uri  if backendSupportsCsvUploader is false and enableRenamingProjectToWorkspace is true", () => {
```

Comment:
Please add one more test case when `enableRenamingProjectToWorkspace` is `false`.

`it("should not return load item with project in uri if backendSupportsCsvUploader is false and enableRenamingProjectToWorkspace is false"`

Reply:
As the existing test can be considered for case when `enableRenamingProjectToWorkspace` is `false`, the comment is marked as resolved.